test/localtest.sh

```diff
@@ -20,7 +20,7 @@ EXPECTED_MINOR_VERSION="${EXPECTED_MINOR_VERSION:-${DD_AGENT_MINOR_VERSION}}"
 if command -v dpkg > /dev/null; then
     apt-get install -y debsums
     debsums -c ${EXPECTED_FLAVOR}
-    INSTALLED_VERSION=$(dpkg-query -f='${source:Upstream-Version}' -W ${EXPECTED_FLAVOR})
+    INSTALLED_VERSION=$(dpkg-query -W ${EXPECTED_FLAVOR} | sed -E "s/[^\t]*\t//g" | sed -E "s/[0-9]+://g")
```

Comment:
I think `cut -f2 | cut -d: -f2` would be more readable than the regexes. Would that work?

Reply:
I'll check if it works the same 👍

Reply:
It should work all the same and is way easier to read :D